include/clang/Interpreter/CppInterOp.h

```diff
@@ -262,16 +264,16 @@ namespace Cpp {
 
   /// This is similar to GetName() function, but besides
   /// the name, it also gets the template arguments.
-  CPPINTEROP_API std::string GetCompleteName(TCppType_t klass);
+  CPPINTEROP_API std::string GetCompleteName(TCppScope_t klass);
```

Comment:
Thanks! We should probably adopt the `CX` enum style so that the compiler could help us with finding these wrong cases. Related to https://github.com/compiler-research/CPyCppyy/issues/8.